contracts/libraries/Pool.sol

```diff
@@ -138,6 +145,12 @@ library Pool {
         self.slot0.hookFees = hookFees;
     }
 
+    /// @notice Only dynamic fee pools may update the swap fee.
+    function _setSwapFee(State storage self, uint24 swapFee) internal {
```

Comment:
just convention but think we should omit the underscore here? since it's called from other locations via library `using for`, think we generally use no underscore. Underscore would be for purely private functions i think

Reply:
ah ok priv funcs makes sense